src/index.css

```diff
@@ -56,26 +60,36 @@ html {
 
 body {
 	background-color: var(--color-bg);
+	font-family: 'Poppins', sans-serif;
 	color: var(--color-text);
-	font-family: -apple-system, BlinkMacSystemFont, avenir next, avenir, segoe ui,
-		helvetica neue, helvetica, Ubuntu, roboto, noto, arial, sans-serif;
 	font-size: 1.8rem;
 	line-height: 1.4;
 	margin: 0;
 
 	-webkit-font-smoothing: antialiased;
 	-moz-osx-font-smoothing: grayscale;
+	scroll-behavior: smooth;
+	/* overflow: hidden;  Disable scrolling */
+}
+
+.btn-neutral {
+	@apply text-2xl btn-lg text-white;
 }
 
-code {
+åcode {
```

Comment:
looks like a typo here

Reply:
Great catch! I've cleaned up and removed those unused CSS